omod/src/main/java/org/openmrs/module/registrationapp/form/RegisterPatientFormBuilder.java

```diff
@@ -54,10 +54,10 @@
  */
 public class RegisterPatientFormBuilder {
 
+	public static final String DEMOGRAPHICS_SECTION_ID = "demographics";
+
 	protected final static Log log = LogFactory.getLog(RegisterPatientFormBuilder.class);
 
-	public static final String DEMOGRAPHICS_SECTION_ID = "demographics";
-		
```

Comment:
![image](https://user-images.githubusercontent.com/9057882/94188936-5c9e6000-feaa-11ea-934d-cadf77b10684.png)
Why did you move the fragment marked 3 to the position marked 1 and left what you should change, which is the position marked 2?

Reply:
I have made the requested changes and for the change of position, I just thought it will be more okay declaring the public first before the protected.  

> ![image](https://user-images.githubusercontent.com/9057882/94188936-5c9e6000-feaa-11ea-934d-cadf77b10684.png)
 Why did you move the fragment marked 3 to the position marked 1 and left what you should change, which is the position marked 2?

Reply:
We try not to modify what is not the subject of the ticket. If something needs to be changed in the code, we try to create a separate ticket for it. ;)